src/cpuid/src/transformer/intel.rs

```diff
@@ -139,6 +163,32 @@ mod tests {
     use crate::cpu_leaf::leaf_0xb::{LEVEL_TYPE_CORE, LEVEL_TYPE_THREAD};
     use crate::transformer::VmSpec;
 
+    #[test]
+    fn test_process_cpuid() {
+        let vm_spec = VmSpec::new(0, 1, false).expect("Error creating vm_spec");
+        let mut cpuid = CpuId::new(0).unwrap();
+        let entry = kvm_cpuid_entry2 {
+            function: leaf_0xb::LEAF_NUM,
+            index: 0,
+            flags: 0,
+            eax: 0,
+            ebx: 0,
+            ecx: 0,
+            edx: 0,
+            padding: [0, 0, 0],
+        };
+        cpuid.push(entry).unwrap();
+
+        assert!(IntelCpuidTransformer {}
+            .process_cpuid(&mut cpuid, &vm_spec)
+            .is_ok());
+        assert!(!cpuid.as_slice().contains(&entry));
```

Comment:
I'm not sure it's in line with the rest of our tests, but a brief description of what is being done in the test and what result is expected would help it read better IMHO.

Reply:
Good point! Additional commit would be helpful and it seems to be able to be simplified.
Thanks!